images/Dockerfile.kairos

```diff
@@ -1,6 +1,8 @@
 # WARNING: Do not build this image on its own, use the different Dockerfile.kairos-* instead
 FROM --platform="linux/${TARGETARCH}" quay.io/kairos/framework:${FRAMEWORK_VERSION}_${SECURITY_PROFILE} AS framework
 
+FROM quay.io/luet/base:0.35.0 AS luet
```

Comment:
This is the only major change, since luet used to be part of the old framework images and now it will be only added until the kairos part of the factory. Any opinions on that?

Reply:
You mean it won't be in the final images anymore right? Hm, I see we shell out to `luet` in the kairos-sdk (`git grep luet`). Maybe things will break if it's not there?

Reply:
Hm no, it's the final image afaict. It should be fine then.

Reply:
ok, moved it back to the framework image